config

```diff
@@ -53,6 +53,24 @@
 [auth]
 
 # Authentication method
+# Value: none | htpasswd | remote_user | http_x_remote_user | ldap
```

Comment:
this is redundant with line 56, please move the following lines below `#type = none`